packages/block-editor/src/components/block-navigation/style.scss

```diff
@@ -1,10 +1,6 @@
 $tree-border-width: 2px;
 $tree-item-height: 36px;
 
-.block-editor-block-navigation__container {
-	padding: $grid-unit-10 - $border-width;
-}
-
```

Comment:
A few lines below this part we set the padding to `padding: $grid-unit - $border-width;` so this rule was always overridden and never applied.